gitbase-playground/templates/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if and .Values.settings.bblfshdServer .Values.settings.gitbaseServer }}
+{{- $port := default "3306" .Values.settings.gitbaseServer.port -}}
+{{- $user := default "gitbase" .Values.settings.gitbaseServer.user -}}
+{{- $database := default "none" .Values.settings.gitbaseServer.database -}}
+{{- $address := required "Missing settings.gitbaseServer.address" .Values.settings.gitbaseServer.address -}}
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "gitbase-playground.fullname" . }}
+  labels:
+    app: {{ template "gitbase-playground.name" . }}
+    chart: {{ template "gitbase-playground.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "gitbase-playground.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "gitbase-playground.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: gitbase-playground
+          image: "{{ .Values.gitbasePlayground.image.repository }}:{{ required "Missing gitbasePlayground.image.tag" .Values.gitbasePlayground.image.tag }}"
+          imagePullPolicy: {{ .Values.gitbasePlayground.image.pullPolicy }}
+          env:
+            - name: PORT
+              value: "{{ .Values.service.internalPort }}"
+            - name: GITBASEPG_ENV
+              value: "{{ .Values.settings.gitbasePlaygroundEnv }}"
+            - name: GITBASEPG_DB_CONNECTION
+            {{- if .Values.settings.gitbaseServer.password }}
+              value: {{ printf "%s:%s@tcp(%s:%s)/%s?maxAllowedPacket=%s" $user .Values.settings.gitbaseServer.password $address $port $database .Values.gitbasePlayground.gitbaseMaxAllowedPacket }}
+            {{- else }}
+              value: {{ printf "%s@tcp(%s:%s)/%s?maxAllowedPacket=%s" $user $address $port $database .Values.gitbasePlayground.gitbaseMaxAllowedPacket }}
+            {{- end }}
+            - name: GITBASEPG_BBLFSHD_URL
```

Comment:
This should be [`GITBASEPG_BBLFSH_SERVER_URL`](https://github.com/src-d/gitbase-playground/blob/master/docs/CONTRIBUTING.md#configuration)

Reply:
fixed in 8a1038b